apps/system/utils/kdbg_env.c

```diff
@@ -76,8 +57,7 @@ int kdbg_env_unset(int argc, char **args)
 	int ret;
 
 	if ((argc != 2) || !strcmp(args[1], "--help")) {
-		kdbg_env_usages(UNSET);
-		return ERROR;
+		goto usage;
```

Comment:
Do you need to use goto? Only here, goto is used. just move them here without goto.